pkg/state/manager.go

```diff
@@ -31,6 +32,8 @@ type Manager interface {
 		templateContext map[string]interface{},
 	) error
 	TryLoad() (State, error)
+	SafeStateUpdate(updater StateUpdate) error
```

Comment:
Wondering if the `Safe` is necessary. Are there still unsafe ways of doing this? Would it make more sense to rename those methods as unsafe to warn potential clients? Or better yet remove those methods entirely?

Reply:
tl;dr I don't think safety should be opt in :)